andela/ah-codeblooded-frontend
# #160577593 Users should be able to like and dislike comments

**What does this PR do?**

Implements like and dislike comment feature.

**Description of Task to be completed?**

- Enables authenticate user to like specific comments
- Enables authenticate user to dislike specific comments

**How should this be manually tested?**

- Log in to the app.
- Navigate /commentss to the address bar and hit enter. 
- You should be taken to a page that displays thumbs up and thumbs down icons

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[160577593](https://www.pivotaltracker.com/story/show/160577593)

**Screenshots (if appropriate)**
<img width="281" alt="screen shot 2018-11-22 at 11 32 58" src="https://user-images.githubusercontent.com/40663486/48890654-77d15c00-ee4a-11e8-9a18-ebddbecb1360.png">
<img width="289" alt="screen shot 2018-11-22 at 11 33 08" src="https://user-images.githubusercontent.com/40663486/48890668-7c961000-ee4a-11e8-845f-6e7cee01ee00.png">


**Questions:**
N/A
